Jbarget/oxymore
# adds country codes to allowed shipping countries

### What changes have you made?
- added country codes to the Stripe shipping `allowedCountries` list 

### Which issue(s) does this PR fix?

Fixes #214 

### Screenshots (if there are design changes)
<img width="378" alt="Screenshot 2020-11-11 at 12 10 58" src="https://user-images.githubusercontent.com/53219789/98804839-fef0b380-2416-11eb-9cc6-b54930e77d75.png">


### How to test
- Click the buy button and go to the Shipping address dropdown to see if all shipping countries display
- Make sure the following unsupported country codes are not in the `allowedCountries` array: 
AS, CX, CC, CU, HM, IR, KP, MH, FM, NF, MP, PW, SD, SY, UM, VI
